rest-api/README.md

```diff
@@ -260,11 +260,58 @@ use Rebuild in CircleCI to retry.
 CircleCI will also open a new JIRA issue to track the release (or update an
 existing issue for cherry picks).
 
-### Tools
+## Descriptions of files and directories
+
+### YAML files
+
+Path|Description
+----|-----------
+[app_base.yaml](app_base.yaml)|Base AppEngine configuration for REST API; gets merged with [app_prod.yaml](app_prod.yaml) for production and [app_nonprod.yaml](app_nonprod.yaml) for non-prod environments
+[app_nonprod.yaml](app_nonprod.yaml)|API AppEngine configuration specific to non-prod environments
+[app_prod.yaml](app_prod.yaml)|API AppEngine configuration specific to production environment
+[cron_default.yaml](cron_default.yaml)|Cron configuration for our Cloud environments (other than sandbox)
+[cron_sandbox.yaml](cron_sandbox.yaml)|Cron configuration for our sandbox Cloud environment
+[offline.yaml](offline.yaml)|AppEngine configuration for offline service
+[queue.yaml](queue.yaml)|Configuration for tasks queues
+[test.yaml](test.yaml)|Combined REST API and offline AppEngine configuration; used for running dev_appserver.py locally
+
+### Entry points
+
+Path|Description
+----|-----------
+[main.py](main.py)|REST API entry point; defines the endpoints exposed in our API
+[offline/main.py](offline/main.py)|Offline entry point; defines the endpoints used to start offline jobs
+
+### Directories in GitHub
+Path|Description
+----|-----------
+[alembic/](alembic)|Schema migrations generated by Alembic (via [tools/generate_schema.sh](tools/generate_schema.sh)) to update our database
+[api/](api)|API classes for our REST API endpoints, referenced by [main.py](main.py)
+[app_data/](app_data)|Data files deployed with AppEngine, to be used by our server (at this point, just code in [data_gen/](data_gen/)
+[appengine-mapreduce/](appengine-mapreduce)|Git submodule containing AppEngine mapreduce code; used by our metrics pipeline (which may soon be retired)
+[config/](config)|Server configuration files; saved into Cloud Datastore and cached by the server
+[dao/](dao)|Data access objects used to read and write to the database
+[data/](data)|Awardee, organization, and site CSV data; extracted from Google Sheets using [tools/export_organizations.sh](tools/export_organizations.sh) and imported into the database using [tools/import_organizations.sh](tools/import_organizations.sh)
+[data_gen/](data_gen)|Code used to generate synthetic data for testing purposes
+[etl/](etl)|ETL SQL and scripts used to create OMOP data in the RDR database
+[etl_test/](etl_test)|SQL that can be used to test ETL logic
+[model/](model)|SQLAlchemy model objects representing entities stored in our database
+[offline/](offline)|Offline jobs that run in a separate service
```

Comment:
Would it be beneficial to mention these are ran as cron jobs ?

Reply:
Done.